src/main/scala/tech/sourced/gemini/QueryApp.scala

```diff
@@ -38,11 +46,23 @@ object QueryApp extends App {
       .action((x, c) => c.copy(feHost = x))
       .text("host is features-extractor server host")
     opt[Int]("features-extractor-port")
-      .action((x, c) => c.copy(bblfshPort = x))
+      .action((x, c) => c.copy(fePort = x))
       .text("port is features-extractor server port")
     opt[Unit]('v', "verbose")
       .action((_, c) => c.copy(verbose = true))
       .text("producing more verbose debug output")
+    opt[String]("doc-freq-file")
+      .action((x, c) => c.copy(docFreqFile = x))
+      .text("path to file with feature frequencies")
+    opt[String]("params-file")
+      .action((x, c) => c.copy(paramsFile = x))
+      .text("path to file with feature frequencies")
+    opt[Int]("hashtables-num")
+      .action((x, c) => c.copy(hashtablesNum = x))
+      .text("number of hashtables should match docFreq file")
+    opt[Int]("band-size")
```

Comment:
I think for these two, `hashtables-num` and `band-size` - let's not to expose though a CLI and just use a hard-coded constants with same defaults as in Apollo (hash).

We going to adjust the defaults later on, based on ML research.

Reply:
this params right now depend on vocab size. While we don't have our own hash we need to be able to pass the same params as apollo calculates. But later yes, we will be able to remove them from CLI.